frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
I already gave the top popsition to sidebar, but to make a required design I have to give that calc height. Please check the design first I think you are missing something.